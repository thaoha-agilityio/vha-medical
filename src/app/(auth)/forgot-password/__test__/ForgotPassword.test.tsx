import { render } from '@testing-library/react';
import ForgotPasswordPage from '../page';

describe('ForgotPassword page test case', () => {
  const setup = () => render(<ForgotPasswordPage />);

  it('should render correctly', () => {
    const { asFragment } = setup();

    expect(asFragment()).toMatchSnapshot();
  });
});
