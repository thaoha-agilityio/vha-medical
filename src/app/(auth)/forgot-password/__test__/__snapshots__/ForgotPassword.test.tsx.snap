// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPassword page test case should render correctly 1`] = `
<DocumentFragment>
  <h2>
    Forgot password page
  </h2>
</DocumentFragment>
`;
